src/components/features/Deals/Deals.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Deals.module.scss';
+
+class Deals extends React.Component {
+  render() {
+    const { products } = this.props;
+
+    const dealsProducts = products.filter(item => item.deals === true);
```

Comment:
Do tego powinien być zrobiony selektor

Reply:
mógłbyś to wyjaśnić, opisać...

Reply:
Wydaje mi się że Kamilowi chodziło o to że selektor do tego może być zrobiony w productRedux.js i stamtąd przekazany do containera.

Reply:
ok, zrobione